server/src/queries/companyQueries.ts

```diff
@@ -0,0 +1,6 @@
+import { db } from "../db";
+import { Company } from "@common/types";
+
+export async function findCompanies(): Promise<Company[]> {
+  return await db.query.CompanyTable.findMany();
+}
```

Comment:
jeśli to jest jedyna asynchroniczna funkcja którą wywołujesz w funkcji, to może być też tak:
```suggestion
export function findCompanies(): Promise<Company[]> {
  return db.query.CompanyTable.findMany();
}
```
dlatego, że ta funkcja tak czy siak zwraca promise'a.

To miałoby znaczenie gdybyś chciał tam zrobić `try catch`, wtedy `async await` byłby potrzebny, żeby złapać błąd przed `return`

https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise